hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -290,70 +287,77 @@ By default, no extra debug messages are enabled.
 XHCI_DEBUG=0x3
 ```
 Default: 0x0  
-Version: pieeprom-2020-06-15.bin - STABLE  
+Version: 2020-09-03  
 
 ## config.txt - configuration properties
-### BOOT_LOAD_FLAGS
+### boot_load_flags
 Experimental property for custom firmware (bare metal).
 
 Bit 0 (0x1) indicates that the .elf file is custom firmware. This disables any compatiblity checks (e.g. is USB MSD boot supported) and resets PCIe before starting the executable. 
 
 Default: 0x0  
-Version: pieeprom-2020-06-15.bin - STABLE  
+Version: 2020-09-03  
 
-## Network Boot
-### Server configuration                                                    
-Network boot requires a TFTP and NFS server to be configured.  See [Network boot server tutorial](bootmodes/net_tutorial.md)
+### uart_2ndstage
+If set to 0x1 then enable debug logging to the UART. In newer firmware versions (Raspberry Pi OS 2020-08-20 and later) UART logging is also automatically enabled in start.elf. This is also described on the [Boot options](../../configuration/config-txt/boot.md) page.
```

Comment:
Might be worth clarifying how this relates to BOOT_UART (is it the same thing?).

The config-txt/boot.md page warns "if the UART is accessed simultaneously to output from Linux then data loss can occur leading to corrupted output" and yet the paragraph here implies it's enabled by default in Raspberry Pi OS 2020-08-20 and later ??

Reply:
Added note, similar but not identical